.github/workflows/ci.yaml

```diff
@@ -23,18 +23,11 @@ jobs:
       matrix:
         os:
           - ubuntu-24.04
-          - windows-2022
-        python-version: ["3.11", "3.12"]
-        is-draft-pr:
-          - ${{ github.event.pull_request.draft }}
+        python-version: ["3.11"]
```

Comment:
note for reviewer: Python 3.11 because that's our min target set in pyproject.toml